scripts/project-setup-scripts/add-folder-roles-for-service-accounts-and-terra-billing-and-test-users.sh

```diff
@@ -54,17 +62,8 @@ RAWLS_CONDITION_FILE="rawls-sa-condition-file.yaml"
 gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=serviceAccount:"${RAWLS_SA}" --role=roles/resourcemanager.projectIamAdmin --condition-from-file=${RAWLS_CONDITION_FILE} # required to set policies on google projects from the Buffer service
 gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=serviceAccount:"${CROMWELL_SA}" --role=roles/editor --condition=None
 
+# Billing and admin group setup
+gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:terra-billing@${APPS_DOMAIN} --role=roles/owner  --condition=None
+gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:firecloud-project-owners@${APPS_DOMAIN} --role=roles/owner  --condition=None
 
-# prod only needs the terra-billing group
-if [[ "${ENV}" == "prod" ]]; then
-  gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:terra-billing@firecloud.org --role=roles/owner  --condition=None
-else # for non-prod envs, add terra-billing as well as project owner groups
-  gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:terra-billing@test.firecloud.org --role=roles/owner  --condition=None
-  gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:firecloud-project-owners@test.firecloud.org --role=roles/owner  --condition=None
-
-  # for the QA env, add quality.firecloud.org users as well
-  if [[ "${ENV}" == "fiab-qa" ]]; then
-    gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:terra-billing@quality.firecloud.org --role=roles/owner  --condition=None
-    gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:firecloud-project-owners@quality.firecloud.org --role=roles/owner  --condition=None
-  fi
-fi
+echo "Done"
```

Comment:
I would probably remove this echo line? It looks new in this, and presumably it just returns to empty terminal if it is done?

Reply:
the output is a bit long and so I thought it would be nice to have a success message at the end. I'll slack you the output to see what you think

Reply:
we decided to go with "Success" from our slack discussion